ionic2/app/api/api.ts

```diff
@@ -0,0 +1,134 @@
+import { Injectable } from '@angular/core';
+import { Http, Response } from '@angular/http';
+import { Observable } from 'rxjs/Observable';
+
+import { Pokemon } from './models/pokemon';
+import { PokemonIcon } from './models/pokemon-icon';
+import { Sighting } from './models/sighting';
+
+export { Pokemon, PokemonIcon, Sighting}
+
+@Injectable()
+export class Api {
```

Comment:
I renamed it to ApiService. I think PokemonService would be too general and the API also returns Sightings and Icons. not just Pokemon.
I also looked into the official [Angular2 style guide](https://angular.io/styleguide#!#naming). According to that guide we should probably also rename the following files/classes:
`App [app.ts]` -> `AppComponent [app.component.ts]`
`Map [map.ts]` -> `MapComponent [map.component.{ts|html|scss}]]`
`About [about.ts]` -> `AboutComponent [about.component.{ts|html|scss}]]`
`Wiki [wiki.ts]` -> `WikiComponent [wiki.component.{ts|html|scss}]]`

What do you think? @WoH